tasks/85_bet-validation/bet-validation.js

```diff
@@ -0,0 +1,19 @@
+// eslint-disable-next-line camelcase
+export const validate_bet = (arrRange, str) => {
+  let pattern = /[^0-9|\s|,]/;
+  if (pattern.test(str)) {
+    return 'None';
+  }
+  let arrStr = str.split(/\s|,/)
+      .filter((cur) => cur !== '').sort();
```

Comment:
here you sort strings but not numbers, what you will have in result if input string will be like this `1,2,3,14`